packages/contracts/token_contract/tests/functions/deposit.rs

```diff
@@ -0,0 +1,78 @@
+use crate::utils::{abi_calls::*, test_helpers::setup};
+use fuels::{prelude::*, types::*};
+
+mod success {
+    use super::*;
+
+    #[tokio::test]
+    async fn can_deposit_to_contract() {
+        let meta = setup(None).await;
+        let token_contract = meta.token_contract;
+        let user = meta.user;
+        let params = meta.params;
+        let provider = meta.provider;
```

Comment:
I don't think that recreating local variables adds any benefits here. I would use the `meta` directly. I would also rename it from `meta` to something more descriptive